src/main/webapp/app/exercises/shared/exercise/_exercise-update.scss

```diff
@@ -16,3 +16,40 @@ label {
     font-size: 17px;
     font-weight: bolder;
 }
+
+.badge {
```

Comment:
not descriptive enough. If moved to a single component, it's okay.

Reply:
This class class just gives a `<span>` the form of badge. `.beta-badge` would not make sense, as color and placement of it is defined by other classes

Reply:
my bad, then